src/components/pages/AppOverview/index.tsx

```diff
@@ -222,7 +222,8 @@ export default function AppOverview() {
 
   const doSearch = useCallback(
     (expr: string) => {
-      const validateExpr = /^[ A-Za-z]*$/.test(expr)
+      const validateExpr =
+        /^[ A-Za-z0-9!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]*$/.test(expr)
```

Comment:
pls move this to Patterns file and and valid and invalid test case for it

Reply:
@manojava-gk updated.